xkcd/cmd/main.go

```diff
@@ -0,0 +1,42 @@
+// xkcd is an xkcd offline indexing and quering tool.
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/vespian/go-exercises/xkcd/pkg/cmdline"
```

Comment:
I am not sure if we can compare stdlib to stdalone app when it comes to structuring your project. Kubernetes is BIG, and personally I find the way they handled this complexity with 'pkg/' folder very appealing. 2cents apply OFC :)